src/RobotInterface/include/BipedalLocomotion/RobotInterface/ISensorBridge.h

```diff
@@ -35,14 +35,16 @@ struct SensorBridgeOptions
     bool isIMUEnabled{false}; /**< flag to connect IMU measurement sources */
     bool isLinearAccelerometerEnabled{false}; /**< flag to connect linear accelerometer measurement sources */
     bool isGyroscopeEnabled{false}; /**< flag to connect gyroscope measurement sources */
+    bool isOrientationSensorEnabled{false}; /**< flag to connect gyroscope measurement sources */
     bool isMagnetometerEnabled{false}; /**< flag to connect magnetometer measurement sources */
     bool isSixAxisForceTorqueSensorEnabled{false}; /**< flag to connect six axis force torque measurement sources */
     bool isThreeAxisForceTorqueSensorEnabled{false}; /**< flag to connect six axis force torque measurement sources */
     bool isCartesianWrenchEnabled{false}; /**< flag to connect cartesian wrench measurement sources */
     bool isCameraEnabled{false}; /**< flag to connect camera sources */
 
     size_t nrJoints{0}; /**< number of joints available through Kinematics stream, to be configured at initialization */
-    std::unordered_map<std::string, std::pair<int, int> > imgDimensions; /**< dimensions of the camera images available through camera streams, to be configured at initialization */
+    std::unordered_map<std::string, std::pair<int, int> > rgbImgDimensions; /**< dimensions of the images available through rgb camera streams, to be configured at initialization */
+    std::unordered_map<std::string, std::pair<int, int> > rgbdImgDimensions; /**< dimensions of the depth images available through rgbd camera streams, to be configured at initialization */
```

Comment:
since the dimension is always a positive number probably you can use `std::size_t`

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-framework/pull/106/commits/0593bcfbf32c11c49d2c9504bbc0aa130d441148